filebeat/input/docker/config.go

```diff
@@ -40,9 +41,12 @@ type config struct {
 }
 
 type containers struct {
+	// IDs + Path to be removed in 8.0 in favor of Paths.
 	IDs  []string `config:"ids"`
 	Path string   `config:"path"`
 
+	Paths []string `config:"paths"`
```

Comment:
Oh, and could you add something to the documentation about the new `paths` option?

Reply:
can this be a follow up PR? I have some more changes to autodiscover that I am planning to submit. I will update docs in a single PR if that is ok.

Reply:
Sure